src/vcastTestInterface.ts

```diff
@@ -362,7 +363,11 @@ export async function getResultFileForTest(testID: string) {
       const firstLineOfOutput: string = commandStatus.stdout
         .split("\n", 1)[0]
         .trim();
+
+      // Delete the REPORT substring and change the .txt to .html because
+      // generate_report only generates one html file and we do not have a .txt
       resultFile = firstLineOfOutput.replace("REPORT:", "");
+      resultFile = resultFile.replace(".txt", "");
```

Comment:
what writes `.txt`?

Reply:
We don't, but the `resultFile` name is saved as a .txt 

https://github.com/vectorgrp/vector-vscode-vcast/blob/2821d34fde0e9e7c1bdc3bd890665cf11241a0fe/src/vcastTestInterface.ts#L355

So we are searching for a `txt` file while only having an `HTML`. That's why I am replacing the "what are we searching for" string.